packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -30,6 +30,12 @@ export class VaFileInput {
   private fileInputRef!: HTMLInputElement;
```

Comment:
This fixes a bug that I discovered where if you remove all the files in the va-file-input-multiple field, the instruction text would not reset and the user would be left with a blank white clickable box.